pkg/controller/migration/convert/cni.go

```diff
@@ -14,6 +14,12 @@ import (
 func loadCNI(c *components) error {
 	cniConfig, err := c.node.getEnv(ctx, c.client, containerInstallCNI, "CNI_NETWORK_CONFIG")
 	if err != nil {
+		if IsContainerNotFound(err) {
```

Comment:
ok, but couldn't you just call `getContainer` first instead of introducing this error type?